javalin/src/main/java/io/javalin/jetty/JavalinJettyServlet.kt

```diff
@@ -43,14 +44,15 @@ class JavalinJettyServlet(val config: JavalinConfig, private val httpServlet: Ja
         wsPathMatcher.add(WsEntry(handlerType, path, config.ignoreTrailingSlashes, WsConfig().apply { ws.accept(this) }, roles))
     }
 
-    override fun configure(factory: WebSocketServletFactory) { // this is called once, before everything
+    override fun configure(factory: JettyWebSocketServletFactory) { // this is called once, before everything
         config.inner.wsFactoryConfig?.accept(factory)
-        factory.creator = WebSocketCreator { req, _ -> // this is called when a websocket is created (after [service])
+        factory.setCreator(JettyWebSocketCreator { req, _ -> // this is called when a websocket is created (after [service])
```

Comment:
Is this something other than a syntax change?

Reply:
setCreator is a setter with various variants, so we can use property accessor anymore